src/app/pages/payment/pin-payment.component.ts

```diff
@@ -0,0 +1,86 @@
+import { validateInput } from './../../validators/payment-validators';
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms'
+import { PaymentService } from 'src/app/services/payment-service';
+import { NgxSpinnerService } from "ngx-spinner";
+import { Router } from '@angular/router';
+import { ToastrService } from 'ngx-toastr';
+
+@Component({
+  selector: 'app-pin-payment',
+  templateUrl: './pin-payment.component.html',
+  styleUrls: ['./payment.component.scss']
+})
+export class PinPaymentComponent implements OnInit {
+	validYears: number[];
+	validMonths: string[] = [
+		'01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12'
+	]
+	cardForm = new FormGroup({
+    cardNumber: new FormControl('5399838383838381', Validators.required),
```

Comment:
Great work @Victoradukwu, I think this default value is misleading, a user might feel that the field is populated already, we could start the value as empty, or add a placeholder like,'Enter card number'.